src/Plugin.php

```diff
@@ -137,12 +137,23 @@ public static function purgePost($postId) {
   /**
    * Registers the updated media file and its derivative image sizes for purging.
    *
+   * Since the file key is added to image attachments only, ensure to register
+   * remaining mime types and possible preview images for purging.
```

Comment:
Please carify in code: Who is supposed to register the mime types and where do they need to be registered?